views/stanford/about_get_involved.pug

```diff
@@ -2,29 +2,126 @@ extends ../layout
 
 block styles
   link(rel='stylesheet', href=Config.ASSET_CDN + '/stylesheets/thingpedia.css')
+  link(rel='stylesheet', href=Config.ASSET_CDN + '/stylesheets/get-involved.css')
+  link(rel='stylesheet', href=Config.ASSET_CDN + '/stylesheets/glide.core.min.css')
+  link(rel='stylesheet', href=Config.ASSET_CDN + '/stylesheets/glide.theme.min.css')
+
 
 block page_name
   - stdlayout = false;
-  = _("Contribute To Almond")
+  = _("Get Involved | Almond")
```

Comment:
Huh?

Reply:
Isn't the page called "Get Involved" now?

Reply:
Why the bar though? If you want a bar and two items in the navbar, wouldn't it added to the navbar instead?

Reply:
Oh, I meant for the page_name to only be displayed as the page title (displayed on the tab). I would prefer the page name not to be displayed in the navbar.